docs/workflows/transitions/preprocess/preprocess/utils.py

```diff
@@ -211,3 +217,27 @@ def patch_empty_predictions(predictions, labels, no_empty_prediction_fields):
     return patched_predictions
 
 
+def filter_away_low_confidence_lines(predictions, field_config):
+    column_names = {
+        label: {column_name for column_name in config['fields']}
+        for label, config in field_config.items() if config.get('type') == 'lines'
+    }
+
+    for prediction in predictions:
+        label = prediction['label']
+        if field_config.get(label, {}).get('type') == 'lines':
+            line_predictions = []
+            for line in prediction['value']:
+                top_1_line = filter_by_top1(line, column_names[label])
+                if not top_1_line:
+                    continue
+                # column names that are not present in the line counts as 0% confidence
+                line_columns_present = {line_dict['label'] for line_dict in top_1_line}
```

Comment:
A suggestion for decreasing verbosity - and also, I don't think `line_dict` is a good name for a prediction - I would prefer `prediction` if not `p`.
```suggestion
                line_columns_present = {p['label'] for p in top_1_line}
```